modules/ocf/manifests/repackage.pp

```diff
@@ -2,22 +2,22 @@
     $package     = $title,
     $recommends  = true,
     $backport_on = '',
-    $dist        = $::lsbdistcodename,
+    $dist        = "${::lsbdistcodename}-backports",
```

Comment:
Nice, I think this will be a good way to do this and make it easier to install from different sources.